dev/archery/archery/integration/tester_csharp.py

```diff
@@ -25,12 +26,20 @@
 
 
 _ARTIFACTS_PATH = os.path.join(ARROW_ROOT_DEFAULT, "csharp/artifacts")
+_BUILD_SUBDIR = "Debug/net8.0"
 
 _EXE_PATH = os.path.join(_ARTIFACTS_PATH,
                          "Apache.Arrow.IntegrationTest",
-                         "Debug/net8.0/Apache.Arrow.IntegrationTest",
+                         _BUILD_SUBDIR,
+                         "Apache.Arrow.IntegrationTest",
                          )
```

Comment:
I'm surprised that this change to _EXE_PATH doesn't seem to have an impact on any other code.